setup.py

```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name="whirlpool_sixth_sense",
-    version="0.17.1",
+    version="0.18.0",
```

Comment:
I choose 0.18.0 for testing - just need to sync this up with HA when your ready for a release